src/libponyc/type/subtype.c

```diff
@@ -769,9 +769,10 @@ static bool is_nominal_sub_structural(ast_t* sub, ast_t* super,
   ast_t* sub_def = (ast_t*)ast_data(sub);
   ast_t* super_def = (ast_t*)ast_data(super);
 
-  // Add an assumption: sub <: super
-  if(push_assume(sub, super, opt))
-    return true;
+  // TODO:
+  // // Add an assumption: sub <: super
+  // if(push_assume(sub, super, opt))
+  //   return true;
```

Comment:
Did you mean to keep these around as comments?

If so, it's not clear to me what is left "TODO" about this - maybe it needs more explanation in the comment (or at least in the commit message)